features/provider/integration/edit.feature

```diff
@@ -11,4 +11,13 @@ Feature: Edit Integration
     Given the service has been successfully tested
     When I go to the service integration page
      And I follow "the analytics section"
-    Then I should be on the provider stats usage page
\ No newline at end of file
+    Then I should be on the provider stats usage page
+
+
+  @javascript
+  Scenario: Change the Authentication method should display the right curl command
```

Comment:
I know we are including the new basic http auth method, but in this test might be worthy to test the query and/or the headers curl command generated. However, that coffee is meant to be migrated to webpack assets and be fully tested with js

Reply:
Done